src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -379,7 +378,8 @@ private async System.Threading.Tasks.Task ProcessIssues(AsyncServerStreamingCall
                 startLine: daemonIssue.StartLine,
                 endLine: daemonIssue.EndLine,
                 startLineOffset: daemonIssue.StartLineOffset,
-                endLineOffset: daemonIssue.EndLineOffset
+                endLineOffset: daemonIssue.EndLineOffset,
+                locations: new List<IAnalysisIssueLocation>()
```

Comment:
I think passing `Array.Empty<IAnalysisIssueLocation>` would avoid creating an object each time. Or change the `AnalysisIssue` to support `locations` being `null`.